site/_apps/viper-news.txt

```diff
@@ -12,12 +12,12 @@ api: 'Hacker News Firebase API'
 hosting: 'Google App Engine'
 authors: 
     - name: 'Andrea'
-lighthouse: '93/100'
-interactive-em: '3.1s'
-interactive-faster-3g: '2.0s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170511_7Q_15882b2c52ed51c614ec707736e52940&run=3
-wpt-em-link: https://www.webpagetest.org/result/170511_AW_b2e2360d4207831ece22ab4210ab9444/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170511_7Q_15882b2c52ed51c614ec707736e52940/
+lighthouse: '73/100'
+interactive-em: '2.4s'
+interactive-faster-3g: '1.7s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170611_DC_6bf5fb102603eb30b3d57dc4c25b480b&run=1
```

Comment:
Reading through the Lighthouse report linked up from WebPageTest:

<img width="976" alt="image" src="https://user-images.githubusercontent.com/110953/27020006-2fef1b00-4ef2-11e7-8373-6cf8ab6f2038.png">

The 512px favicon check for the Android splashscreen is a new check in Lighthouse 2.0. @WebReflection any chance you could address that upstream?

There's something strange going on with the redirects for the `responds with a 200 when offline` in the report. 

I'm unsure if the implementation is redirecting the 200s in the offline check, but I've kicked off another run in https://www.webpagetest.org/result/170612_Y5_da09ea5ebe445adbf698b0e1b1b2a5a3/ as a sanity check.